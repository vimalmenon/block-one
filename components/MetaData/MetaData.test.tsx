import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { MetaData } from '.'

describe('MetaData Component', () => {
  test('MetaData: Snapshot testing with title and Description', () => {
    const wrapper = shallow(<MetaData title="title" description="description" />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
  test('MetaData: Snapshot testing with title', () => {
    const wrapper = shallow(<MetaData title="title" />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
