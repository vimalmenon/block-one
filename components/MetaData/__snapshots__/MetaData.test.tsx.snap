// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MetaData Component MetaData: Snapshot testing with title 1`] = `
<Head>
  <title>
    title
     - 
    Fictitious Bank
  </title>
  <meta
    content="This is fictitious bank application"
    name="description"
  />
</Head>
`;

exports[`MetaData Component MetaData: Snapshot testing with title and Description 1`] = `
<Head>
  <title>
    title
     - 
    Fictitious Bank
  </title>
  <meta
    content="description"
    name="description"
  />
</Head>
`;
